internal/gateway/usecases/get_auth_token.go

```diff
@@ -71,6 +72,15 @@ func (s *getAuthTokenUsecase) Run(ctx context.Context) error {
 	}
 	userInfo := metadataManager.GetUserInformation()
 
+	if userInfo.NotBefore.IsZero() {
+		s.Log.Info("User's authentication time could not be determined", "id", userInfo.Id, "name", userInfo.Name, "email", userInfo.Email)
+		return domainErrors.ErrUnauthenticated
+	}
+	if time.Now().UTC().Sub(userInfo.NotBefore) > 1*time.Minute {
```

Comment:
Is 1 minute deviation somehow a default or is this basically a random value?

Reply:
Hm, I am just not sure anymore what I am checking there. 🤔 Will try to make sense out of it.

Reply:
Okay, this is just to make sure the authentication of the user requesting a new token for K8s authentication is new and shiny. So one minute is kind of random. It could be 30s too, but getting smaller here makes it more likely that the request is rejected because some delay between the login and the request for the K8s auth token. In general that is not the greatest solution IMHO. I will try to think about that again.

Reply:
I was wondering whether there's some gap here that could be used for an attack or so. Also not entirely sure. Maybe @cypherfox has an idea ;-)

Reply:
I am sure that is not a gap one could use against m8, but it is kind of a weird way to achieve the goal of having short expiring k8s tokens and force an authentication flow. :D

Reply:
for now i am alright with it.